src/stirling/bpf_tools/bcc_wrapper_bpf_test.cc

```diff
@@ -169,6 +170,52 @@ TEST(BCCWrapperTest, GetTGIDStartTime) {
   EXPECT_EQ(proc_pid_start_time, expected_proc_pid_start_time);
 }
 
+TEST(BCCWrapperTest, GetTGIDStartTime) {
+  // Force the TaskStructResolver to run,
+  // since we're trying to check that it correctly gets the task_struct offsets.
+  std::vector<std::string> cflags = {};
+  bool requires_linux_headers = true;
+  bool always_infer_task_struct_offsets = true;
+
+  BCCWrapperImpl bcc_wrapper;
+  ASSERT_OK(bcc_wrapper.InitBPFProgram(get_tgid_start_time_bcc_script, cflags,
```

Comment:
This doesn't seem like it's attaching the correct BPF program. We should have an additional `OBJ_STRVIEW(get_current_cgroup_id)` and pass in `get_current_cgroup_id_bcc_script`.